src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -356,10 +278,136 @@ protected View DefaultNavigationContent
             get
             {
                 //TODO: Do not set default navigation content if there is no previous page.
+                if (defaultNavigationContent == null)
+                {
+                    defaultNavigationContent = CreateDefaultNavigationContent();
+                }
+
                 return defaultNavigationContent;
             }
         }
 
+        /// <summary>
+        /// Default title content of AppBar set automatically by default.
+        /// If TitleContent is not set by user, then default title content is
+        /// automatically displayed.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected View DefaultTitleContent
+        {
+            get
+            {
+                if (defaultTitleContent == null)
+                {
+                    defaultTitleContent = CreateDefaultTitleContent();
+                }
+
+                return defaultTitleContent;
+            }
+        }
+
+        /// <summary>
+        /// Default action content of AppBar set automatically by default.
+        /// If ActionContent is not set by user, then default action content is
+        /// automatically displayed.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected View DefaultActionContent
+        {
+            get
+            {
+                if (defaultActionContent == null)
+                {
+                    defaultActionContent = CreateDefaultActionContent();
+                }
+
+                return defaultActionContent;
+            }
+        }
+
+        /// <summary>
+        /// Sets title text of AppBar.
+        /// SetTitle sets title text to the default title content.
+        /// Therefore, if TitleContent is set by user, then SetTitle does not set title text of the replaced TitleContent.
+        /// </summary>
+        /// <param name="string">Title text of AppBar.</param>
```

Comment:
could you resolve the warning at 333 line and 335 line below?
it is about comment stuff.
thank you.

Reply:
Thank you for your review.
The param name has been fixed.